samples/tls/logging.md

```diff
@@ -72,29 +79,40 @@ information about creating this secret manually.
 
 ## Specifying TLS for Ingress
 
-If you are using ingress and want to enable TLS on traffic into the cluster,
-you must manually create two Kubernetes secrets.
+If you are using ingress and want to enable TLS on communications into the cluster,
+you must manually create the following two Kubernetes secrets:
+
+- `elasticsearch-ingress-tls-secret`
+- `kibana-ingress-tls-secret`
 
-1. After you have obtained the certificates for the secrets, use this command
-to generate the secrets:
+**Note:** The process of generating the TLS certificates for these secrets is outside the scope for this example.
+
+1. After you have obtained the TLS certificates for each secret, use the following command to generate the secrets. You must run the command for each secret name.
 
 ```bash
-kubectl create secret tls $SECRET_NAME -n $NAMESPACE --key <cert-key> --cert <cert-file>
+kubectl create secret tls $SECRET_NAME -n $NAMESPACE --key $CERT_KEY --cert $CERT_FILE
```

Comment:
I think we need to be consistent with how we present commands and syntax.  The use of a leading dollar sign  and an all caps label (e.g. $FOO) suggests a reference to an environment variable.  The use of some other syntax (enclosing a string w/in < and >, for example, or using italics) seems better for indicating where a user/reader is expected to substitute a value in.  

In this particular case, are there environment vars called CERT_KEY and CERT_FILE that contain the names of the two files needed?  Nothing in our write-up suggests (so far) that's the case.  Hmmm, now I see these vars are "explained" below, so I guess the syntax is correct.  But I'm not sure I like this.  I will defer to others.

Reply:
I belated came across comparable commands in the Help center content. I will adjust these commands to use the Help center syntax.

Reply:
I reviewed the following articles about "kubectl create secret tls":

- https://docs.microsoft.com/en-us/azure/aks/ingress-own-tls 
- https://kubernetes.io/docs/concepts/configuration/secret/ 
- https://blog.knoldus.com/how-to-create-a-tls-certificate-for-ingress-in-kuberentes/

It appears that the syntax should be: 
kubectl create secret tls _secret_name_ --namespace _namespace_ --key=_cert_key_ --cert=_cert_file_

Please let me know if this adjustment is OK to make.

Reply:
I am fine with the "adjustment";

Reply:
I'd use cert_key_file and cert_file - they're both files (.key and .crt)

Reply:
I have adjusted the syntax as proposed. I did use cert_key_file and cert_file.